adaptors/collections.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Collections Adaptor
+---
+
+## Collections Overview
+
+The Collections API is a key/value storage solution. It is designed for high
```

Comment:
We need to make clear that this is a data store _in_ OpenFn... not something external (like all other adaptors).